.github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] Please make sure to add a good description to your pull request 
- [ ] Please the title of the pull request should describe a feature like feature/addHeading 
![image](https://user-images.githubusercontent.com/74744787/184977049-6615d21b-2470-43e4-81f5-1678f2ed893e.png)

Reply:
Hi,
Thank you.
All changes have been effected.